pkgs/tools/typesetting/asciidoctor/rebuild-gemset.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+  pkgs.mkShell {
+# TODO: how to reuse the list from default.nix?
+    buildInputs = with pkgs; [
```

Comment:
I'd think with #62971 this would be uneeded

Reply:
sorry, not quite sure what you mean :) do you mean this file, or this list here?

any chance you could update the PR to the preferred approach? (please also see my other comment on the PR where I failed to generate the gemset and Gemfile.lock in any other approach I've tried... )